core/src/main/scala/org/apache/spark/util/collection/ExternalSorter.scala

```diff
@@ -113,11 +114,21 @@ private[spark] class ExternalSorter[K, V, C](
     if (shouldPartition) partitioner.get.getPartition(key) else 0
   }
 
+  private val metaInitialRecords = conf.getInt("spark.shuffle.sort.metaInitialRecords", 256)
+  private val kvChunkSize = conf.getInt("spark.shuffle.sort.kvChunkSize", 8192)
```

Comment:
Yeah, I definitely don't think we should expect to tune this in normal operation.  My thinking was not to document it and mainly to make it accessible if people want to play around with it for benchmarking, but I can take it out too.